docs/tutorials/ethereum/exploring-ethereum/map_block_meta_module.md

```diff
@@ -0,0 +1,127 @@
+## Mapping Blocks
+
+This module takes a raw Ethereum block and returns a reduced version of a block, with just three pieces of information: hash, parent hash, and block number.
+
+Let's run the Substreams first, and then go through the code.
+
+### Running the Substreams
+
+Running a Substreams usually requires three steps: generating the Rust Protobufs, building the WASM container, and using the Substream CLI to start the streaming. Make sure to run the following commands in the `substreams-explorer/ethereum-explorer` folder:
+
+1. **Generate the Protobuf objects:** The `.proto` files define a data model regardless of any programming language. However, in order to use this model in your Rust application, you must generate the corresponding Rust data structures.
+
+```bash
+$ make protogen
+```
+
+2. **Build the WASM container:** The following command generates a WASM container from the Rust application, which you can find at `/target/wasm32-unknown-unknown/release/substreams.wasm`. Note that this is the same path provided in the Substreams manifest (`substreams.yml`).
+
+```bash
+$ make build
+```
+
+3. **Streaming data through the CLI:** The following command streams the Ethereum blockchain data, and applies the transformations contained in the `map_block_meta` module to every block.
+
+```bash
+$ substreams run -e mainnet.eth.streamingfast.io:443 substreams.yaml map_block_meta --start-block 17712040 --stop-block +1
+Connected (trace ID e98f04cd7ebb6191befbbf2e6668eafc)
+Progress messages received: 0 (0/sec)
+Backprocessing history up to requested target block 17712040:
+(hit 'm' to switch mode)
```

Comment:
We can probably elide some of that here.. from `Connected` to `Backprocessing`.. we can make it [...]. Less pixels is better.